oauth2-test/src/main/java/org/springframework/security/test/oauth2/support/JwtAuthenticationBuilder.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2002-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.security.test.oauth2.support;
+
+import static org.springframework.security.test.oauth2.support.CollectionsSupport.putIfNotEmpty;
+
+import java.time.Instant;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.security.oauth2.jwt.Jwt;
+import org.springframework.security.oauth2.jwt.JwtClaimNames;
+import org.springframework.security.oauth2.server.resource.authentication.JwtAuthenticationToken;
+
+/**
+ * @author Jérôme Wacongne &lt;ch4mp@c4-soft.com&gt;
+ * @since 5.2.0
+ */
+public class JwtAuthenticationBuilder<T extends JwtAuthenticationBuilder<T>> extends AbstractAuthenticationBuilder<T> {
+
+	public static final String DEFAULT_TOKEN_VALUE = "test.jwt.value";
```

Comment:
Let's not make these public until there is a clear need. This aligns with `UserExchangeMutator` and `UserRequestPostProcessor`.

Reply:
Actually, there is a clear need: it is used by both Jwt request post-processor and reactive server configurer which are in different packages

Reply:
I guess I'm not seeing usage of `DEFAULT_TOKEN_VALUE` in the code, outside of this class. What am I missing? By "these", I meant "these public static final fields".

Reply:
oups, sorry, thought you where talking about JwtAuthenticationBuilder

Done